src/caret_analyze/value_objects/callback.py

```diff
@@ -510,14 +572,41 @@ def __init__(
 
     @property
     def callback_type(self) -> CallbackType:
+        """
+        Get callback type name.
+
+        Returns
+        -------
+        CallbackType
+            callback type
```

Comment:
```suggestion
            Callback type.
```

Reply:
Fixed it in [1cec329](https://github.com/rokamu623/CARET_analyze/pull/24/commits/1cec3298f259204f4eefbf122e2412e3903809fb).